fastlane/Fastfile

```diff
@@ -355,11 +355,15 @@ platform :ios do
     ENV['REPLACE_ASSETS'] = 'true'
     ENV['BUILD_FOR_RELEASE'] = 'true'
     ENV['APP_SCHEME'] = 'mattermost'
-    data_path = 'build-ios'
 
     update_identifiers
     replace_assets
 
+    data_path = 'ios'
+    if ENV['CIRCLECI'] == 'true'
+      data_path = 'build-ios'
+    end
+
     sh 'rm -rf ../build-ios/'
```

Comment:
shouldn't this be `data_path`?  same on L371

Reply:
Honestly, I'm not sure if we should be removing the generated folder/files at `/ios/Build` or `/ios/DerivedData`. I thought no need but let me know if I need to.

Reply:
as far as I know those two that you mentioned there is no need to remove, but `build-ios` for example, if you don't delete it, building  a second consecutive time it will fail.

Reply:
Okay, so I think the current change is fine since it's removing `/build-ios` both in local and CI. In local where `data_path = 'ios'`, it will do nothing anything under `/ios` folder.